### DISCUSSION TITLE
Why do my parser sources not show up in source overview?

### DISCUSSION THREAD

**Discussion by MaxEtMoritz** [openmensa/openmensa#815] [Q&A] [answered]
I recently created a new parser and registered it to OpenMensa.
The parser provides several canteens (over 100). When registering it and hitting "Update sources via Index URL" (Aktualisiere Quellen mittels Index-URL), the server recognized that there are over 100 new sources.
But why do these sources not show up in the source state (Quellenstatus) panel of the parser, not even after a page refresh?
Is there something wrong with my index file?
Here are my [index.json](https://maxetmoritz.github.io/openMensaMax/index.json) and the [Feed XMLs](https://maxetmoritz.github.io/openMensaMax/).

The reason i ask is that i would very much like to avoid having to manually copy-paste the source name and Meta URL for each of these over 100 canteens. Googling them to find out the canteen's address and sometimes even the canteen name for all of them is already enough work&hellip;

**Comment by cvzi** [comment]
No idea if it matters but the obvious difference to my parsers is that there are slashes in the json keys of your index.json

-- 
Enviado desde mi dispositivo Android con K-9 Mail. Por favor, disculpa mi brevedad.

**Reply by MaxEtMoritz** [reply]
And for yours, it shows new sources after syncing with the index.json?
By The way: Thank you very much for the inspiration of running parsers as a GitHub Action 👍🏻

**Reply by cvzi** [reply]
Normally it does show new sources. I haven't checked recently though.

**Comment by MaxEtMoritz** [comment]
Or maybe the parser's meta feed has to provide a name and a city at the minimum for the sources to show up...
The MensaMax plan sadly does not provide this info (not even meal prices🤣)

**Comment by jgraichen** [comment]
I'll try to take a look at the server side soon, but I only have limited time available until next year. Do you have your feed/index/parser ID at hand (should be in the URL)?

**Reply by MaxEtMoritz** [reply, marked as answer]
Thank you for willing to take a look.
Parser ID: 104
lowest source ID: 1623
highest source ID: 1652 (there may of course be sources of other parsers in between)
Lowest Feed ID: 2737
Highest Feed ID: 2766 (there may be feeds of other parsers in between)
Don't know how to get the Index ID though.

~When you find the time, could you also check why the Feed URLs are not updated? I re-organized the file structure of the XMLs in my parser and renamed the sources in the Parser and OpenMensa. Syncing the new Meta URLs from the index.json worked fine, but when trying to update the Feed URLs by syncing feeds with the meta file, it only shows 0 feeds added / updated / deleted and the existing feeds still point to the old file path.~

~Will try and increment the \<version> of the parser to see if that changes anything. if that doesn't help, i will try to rename the feeds in the meta XML (to hopefully make OpenMensa drop the old ones and add the new ones).~

~Edit: Neither of these two worked. so there's nothing but waiting and hoping (or keeping the previous xml file structure)~

Edit 2: Found out why the Feed URLs did not update: my new XML file names included a space, and i typed in / submitted in XML a space instead of %20 🤦🏻‍♂️
